File: Stack/StackQueue.java
 class xyz
 {   Queue<Integer> q1 = new LinkedList<Integer>();
    
    /*The method pop which return the element poped out of the stack*/
    int pop()
    {
	    if (q1.isEmpty())
	        return -1;
	    int size = q1.size();
	    for (int i = 0; i < size - 1; i++)
	        q1.add(q1.poll());
	    return q1.poll();
    }
	
    /* The method push to push element into the stack */
    void push(int a)
    {
	    q1.add(a);
    }
 }

File: Stack/StackQueue.cpp
void QueueStack :: push(int x) {
    q1.push(x);
}
/*The method pop which return the element poped out of the stack*/
int QueueStack :: pop()
{
    if (q1.empty())
        return -1;
    int *back = &q1.back();
    
    while (!q1.empty() && &q1.front() != back) {
        q1.push(q1.front());
        q1.pop();
    }
    int res = q1.front();
    q1.pop();
    return res;
}
